### ISSUE TITLE
pm2 restart sets watching to active

### ISSUE THREAD

**Issue by papakai** [Unitech/pm2#2327]
If reporting a bug, please use the following template.

#### Expected behaviour

Not to enable the watch mode as the json config file has 'watch' : false set!

#### Actual behaviour

Watch mode is activated even though in json config file it is deactivated and when starting the app it then also not activated (nor did we enable it after start)!
When restarting using just the appname (pm2 restart TLDINFO), won't activate the watch mode.

#### Steps to reproduce
Have a json config file where watch: false is set as option. Here our example file:
```
{
  "apps": [{
    "name": "TLDINFO",
    "script": "app.js",
    "error_file": "/var/log/frontend/TLDINFO/err.log",
    "out_file": "/var/log/frontend/TLDINFO/out.log",
    "instances": 1,
    "cwd": "/opt/ftldinfo",
    "exec_mode": "cluster",
    "exec_interpreter": "node",
    "merge_logs": true,
    "log_date_format": "YYYY-MM-DD HH:mm:ss Z",
    "autorestart": true,
    "restart_delay": "10000",
    "watch": false
  }]
}
```
pm2 start <json_file>
pm2 restart <json_file>

#### Software versions used

```
OS         : Debian GNU/Linux 8
node.js    : 4.4.7
PM2        : 2.0.4 (development branch and also pm2@next are affected!)
```

#### PM2 Log output

Use the command: `tail --lines 50 ~/.pm2/pm2.log`

```
2016-08-10 14:35:52: Starting execution sequence in -cluster mode- for app name:TLDINFO id:5
2016-08-10 14:35:52: App name:TLDINFO id:5 online
2016-08-10 14:36:01: [Watch] Start watching 5
2016-08-10 14:36:01: Stopping app:TLDINFO id:5
2016-08-10 14:36:01: App name:TLDINFO id:5 disconnected
2016-08-10 14:36:01: App [TLDINFO] with id [5] and pid [28262], exited with code [0] via signal [SIGINT]
2016-08-10 14:36:02: Starting execution sequence in -cluster mode- for app name:TLDINFO id:5
2016-08-10 14:36:02: App name:TLDINFO id:5 online
```

cf CONTRIBUTING.md for more informations <-- no idea what this means :)

If you need any further information, feel free to ask!

**Comment by Unitech** [comment]
Thanks for reporting @papakai, issue fixed

**Comment by Unitech** [comment]
published under pm2@next (2.0.7)
